stock_move_actual_date/readme/DESCRIPTION.rst

```diff
@@ -0,0 +1,12 @@
+This module adds an Actual Date field to the stock picking, stock scrap,
+stock move, and stock move line models. This field allows users to
+record the actual date on which a stock transfer or stock scrap took
+place, in case the transaction in Odoo is processed after the fact. It
+also adds an Actual Date field to the Stock Valuation Layer model,
```

Comment:
```suggestion
place, in case the transaction in Odoo is processed after the fact.

It also adds an Actual Date field to the Stock Valuation Layer model,
```